operational/vmware/instance_tag_sync/instance_tag_sync.pt

```diff
@@ -0,0 +1,452 @@
+name "VMWare Instance Tag Sync"
+rs_pt_ver 20180301
+type "policy"
+short_description "Adds tags to vmware instances from CMP. See the [README](https://github.com/rightscale/policy_templates/tree/master/operational/vmware/instance_name_sync) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "medium"
+category "Operational"
+
+###############################################################################
+# Permissions
+###############################################################################
+
+permission "perm_read_creds" do
```

Comment:
Is this permission even used?

Reply:
I checked it isn't. I think I was originally going to read some stuff from credentials. I removed it.